rust-toolchain.toml

```diff
@@ -1,4 +0,0 @@
-[toolchain]
```

Comment:
@BudgieInWA, you added this in #23. I'm getting rid of it because:
1) Pulling in `driving-side` created duplicate geo and geojson dependencies
2) I tried `cargo update -p geom` to grab a newer version of the A/B Street crate
3) But hit https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#workspace-inheritance, since I switched to workspace-level dependencies over there.
I realized my Rust beta install was still on 1.62. I've tested everything in osm2streets with 1.64, the latest stable, and it all works. I don't remember what beta features you were making use of a few months ago, but looks like they're in stable now!

Reply:
I think it was some part of the `trunk` that needed beta, so I'm happy to see a return to stable.